docs/configuration.md

```diff
@@ -1664,6 +1664,14 @@ Apart from these, the following properties are also available, and may be useful
         </td>
     </tr>
     <tr>
+        <td><code>spark.ssl.<particular protocol>.port</code></td>
```

Comment:
Yes, setting `spark.ssl.port` sets the SSL port for all services. That's how it's documented to work above, and you can override with `spark.ui.ssl.port`, etc. I'm just saying that this is already documented, and to be consistent with other props, the config key in this section can be `spark.ssl.port` -- or did I misunderstand?